packages/cdktf-cli/test/get/generator/module-generator.test.ts

```diff
@@ -6,7 +6,7 @@ import { TerraformModuleConstraint  } from '../../../lib/config'
 import { expectModuleToMatchSnapshot } from '../util';
 
 test('generate some modules', async () => {
-  jest.setTimeout(20000)
+  jest.setTimeout(60000)
```

Comment:
Mostly out of curiosity: Do you know why the timeout has to be that high?

Reply:
I don't. But I had other tests failing with 40s so I figured to just give it a bit more.
But I'm happy to "downgrade" it.

Reply:
All good 👍